src/components/PoolTransactionsList.tsx

```diff
@@ -0,0 +1,109 @@
+import { useEffect } from "react";
+import { useDispatch } from "react-redux";
+import moment from "moment";
+import { Heading4, TextLink } from "@stellar/design-system";
+import { SortableTable } from "components/SortableTable";
+import { STELLAR_EXPERT_TX } from "constants/apiUrls";
+import { fetchPoolTransactionsAction } from "ducks/poolTransactions";
+import { formatAmount } from "helpers/formatAmount";
+import { useRedux } from "hooks/useRedux";
+import {
+  LiquidityPoolTransaction,
+  LiquidityPoolReserve,
+  ActionStatus,
+} from "types/types.d";
+
+export const PoolTransactionsList = ({ poolId }: { poolId: string }) => {
+  const { poolTransactions } = useRedux("poolTransactions");
+  const dispath = useDispatch();
+
+  useEffect(() => {
+    dispath(fetchPoolTransactionsAction(poolId));
+  }, [poolId, dispath]);
+
+  const labels = [
+    {
+      id: "id",
+      label: "Transaction",
+      sortBy: true,
+    },
+    // TODO: using shares for now. We probably need to fetch current prices to
+    // calculate total value.
+    {
+      id: "shares",
+      label: "Shares",
+      sortBy: true,
+    },
```

Comment:
Keeping it simple for now. Should be pretty straightforward to update to get total value.